### PR TITLE
Try fix sporadic exception in cluster mode

### PR DESCRIPTION
**Pull request by soulasuna**
## Try fix sporadic exception in cluster mode

Related #19103.

Changes proposed in this pull request:
- Try fix sporadic exception in readwrite_splitting  scenario.

### CODE REVIEW

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ddl/BaseDDLIT.java [hunk 1]
```diff
@@ -59,6 +61,14 @@ public final void init() throws Exception {
         try (Connection connection = getTargetDataSource().getConnection()) {
             executeInitSQLs(connection);
         }
+        intervalTime();
+    }
+    
+    @SneakyThrows(InterruptedException.class)
+    private void intervalTime() {
+        if ("Cluster".equalsIgnoreCase(getMode())) {
+            TimeUnit.MILLISECONDS.sleep(200);
```
Comment (terrymanu) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ddl/BaseDDLIT.java:70]: long value should use `L`, please change it to `200L`

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ddl/BaseDDLIT.java [hunk 2]
```diff
@@ -59,6 +61,14 @@ public final void init() throws Exception {
         try (Connection connection = getTargetDataSource().getConnection()) {
             executeInitSQLs(connection);
         }
+        intervalTime();
+    }
+    
+    @SneakyThrows(InterruptedException.class)
+    private void intervalTime() {
```
Comment (terrymanu) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ddl/BaseDDLIT.java:68]: the method name is not clear, method name should be Verb + Nouns

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ddl/BaseDDLIT.java (hunk 1 above)
Comment (soulasuna) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ddl/BaseDDLIT.java:70]: I will fix it.

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ddl/BaseDDLIT.java (hunk 2 above)
Comment (soulasuna) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ddl/BaseDDLIT.java:68]: I will fix it.

**Review by terrymanu — APPROVED** [review]

**Pull request merged**